src/compose/ComposeText.js

```diff
@@ -71,6 +82,30 @@ class ComposeText extends React.Component {
     });
   }
 
+  showErrorAlert = (message, title) => {
+    Alert.alert(
+         title,
+         message,
+         [{ text: 'OK', onPress: () => {} }],
+         { 'cancelable': true } // eslint-disable-line spellcheck/spell-checker
+       );
+  }
+
+  handleEdit = async () => {
+    const { auth, editMessage, cancelEditMessage } = this.props;
+    if (editMessage.content.raw_content === this.state.text) {
+      this.showErrorAlert('No changes detected', 'Cannot edit');
+      cancelEditMessage();
+      return;
+    }
+    try {
+      await patchMessage(auth, this.state.text, editMessage.id);
+    } catch (err) {
+      this.showErrorAlert(err.message, 'Error Occurred');
```

Comment:
We need a better error message.
If it is a general error I suggest 'Failed to edit message' or 'Error while updating message'.